starfish/test/image/test_imagestack_labeled_indices.py

```diff
@@ -31,18 +31,18 @@ def fill_value(round_: int, ch: int, z: int) -> float:
     return float((((round_idx * NUM_CH) + ch_idx) * NUM_Z) + z_idx) / (NUM_ROUND * NUM_CH * NUM_Z)
 
 
-def x_coordinates(round_: int, ch: int) -> Tuple[float, float]:
+def x_coordinates() -> Tuple[float, float]:
     """Return the expected physical x coordinate value for a given round/ch tuple.  Note that in
```

Comment:
I think we could foresee small variations across channel values in the cases where the filters aren't aligned -- BaristaSeq has this problem.

Reply:
?? variations that make the imagestack unaligned?

Reply:
Assuming the filter you are talking about is the filter in front of the sensor to read the different color channels: why would it matter?  Wouldn't the x-y coordinates be just read off of the microscope stage position?  Surely it's not taking the stage position and adding the relative offset from the edge of the sensor to the first pixel that can read a particular channel...?

Reply:
cc: @kevinyamauchi